tests/configuration/pages/pmmInventoryPage.js

```diff
@@ -126,11 +134,18 @@ module.exports = {
     return serviceIds;
   },
 
-  async checkAgentOtherDetailsSection(detailsSection, expectedResult, serviceName, serviceId) {
-    const locator = locate('span').withText(detailsSection).after(locate('span').withText(`service_id: ${serviceId}`));
-    const details = await I.grabTextFrom(locator);
+  async checkAgentOtherDetailsSection(agentType, expectedResult, isDisplayed = true) {
```

Comment:
agentType argument requires explanation and documentation. Where does it come from and what values are accepted. 
Ideally it should be "enum" type but we should use any JS alternative for it. By now I cannot understand how to use it to fix PMM-T1559 test. And I don't know which type should be used https://www.screencast.com/t/lHmeJ6r9o

Reply:
Agent type can be Every type for all databases we have available. So it can vary for each database, so even with enum. You still can put invalid value there.

Reply:
Yes, this is exact reason why I'm asking this change. Because it's not obvious where to get the proper line. The project "common" way to see the agent types is pmm-admin list which has different format, see: https://www.screencast.com/t/SLgj2OrGfJ That is why we need to include minimal explanation what is that sting and where to obtain it. Ideally it should be forced to enum constant.